pygit2/_pygit2.pyi

```diff
@@ -496,7 +496,7 @@ class Repository:
     def revparse_single(self, revision: str) -> Object: ...
     def set_odb(self, odb: Odb) -> None: ...
     def set_refdb(self, refdb: Refdb) -> None: ...
-    def status(self) -> dict[str,int]: ...
+    def status(self, untracked_files: str = "all", ignored: bool = True) -> dict[str,int]: ...
```

Comment:
In https://book.git-scm.com/docs/git-status `--ignored` is actually a string and can take three values:
- "traditional": Shows ignored files and directories, unless --untracked-files=all is specified, in which case individual files in ignored directories are displayed.
-  "no": Show no ignored files
- "match": Shows ignored files and directories matching an ignore pattern.

I'd rather avoid implementing `match`, but I could make `ignored` a `str` kwarg, raising `NotImplementedError` for `match`

Reply:
I have no strong opinion here, so I'm open to whatever you prefer here

Reply:
It's ok as bool